src/constants/gravityObjects/GravityObjects.ts

```diff
@@ -0,0 +1,80 @@
+export class GravityObject {
+  readonly name:string;
+  constructor(name?:string) {
+    this.name = name;
+  }
+
+  toString(): string {
+    return this.name;
+  }
+}
+
+export class AutoGravity {
```

Comment:
Why is that located under `constants/...`?

Reply:
It's a very good question (and part of my notes to the reviewer).

Imports in JS are not like imports in Java or PHP, we have to explicitly import it from somewhere, we don't have namespaces.


There's simply no logical place to put AutoGravity, as there are no similar Entities.
For example, if this was part of Gravity, the code would look like this:

`Gravity.auto(Gravity.AutoGravity.object(GravityObjects.CAT))` - Not ideal.

Reply:
I suggest that things that are related to gravity will be in a Gravity folder, in folders like params, gravityObjects etc

Reply:
That would indeed be great, but how would a user import it? 
Unlike JAVA we can't just import the 'Gravity' namespace.

We also need to support granular imports, a user should only be able to pull only the constants he wants.
It's a lot of things that really limit how flexible we can be with the file structure.

How do you see the whole solution?

Reply:
Sure that there are not other places that such animals exists? Something that will justify creating a separated folder  structure for? 
I would never look for it under `constants`